### PR TITLE
Define "perennial" in docs

### PR DESCRIPTION
**Pull request by allewun**
@kevgo @charlierudolph It's an uncommon word, the definition helps to understand what it is. Also fixed some typos I found.

### CODE REVIEW

**Comment by allonsy** [comment]
👍

**Comment by charlierudolph** [comment]
:ship:

File: src/helpers/git_helpers/branch_configuration_helpers.sh [hunk 1]
```diff
@@ -140,7 +140,7 @@ function echo_update_child_branches {
 # Makes sure that we know all the parent branches
 # Asks the user if necessary
 function ensure_knows_parent_branches {
-  local branches=$1 # space seperated list of branches
+  local branches=$1 # space separated list of branches
```
Comment (kevgo) [inline src/helpers/git_helpers/branch_configuration_helpers.sh]: I can't believe we mistyped this 4 times!!!  :)

**Comment by kevgo** [comment]
Nice! 🚢

**Pull request closed without merging**